### ISSUE TITLE
dd rom support

### ISSUE THREAD

**Issue by LuEnCam** [zeroKilo/N64LoaderWV#24]
I was wondering if this module could support in future reversing dd roms.

Would it be possible ? Or would it be too complicated ?

**Comment by zeroKilo** [comment]
Well I dont have such a rom, so can not test it. Do you have any?

**Comment by LuEnCam** [comment]
Yes I have them. There are 2 different formats : 

1. *.d64 : it's the original format of dd roms. They were build like that with the compiler 
2. *.ndd : it's the format that we mainly find on the internet. It's the format that was used to dump the roms from the disk drive with a flashcart. 

All dd roms can be found here : https://64dd.org/games.html . If you would like to see the .d64 formats, let me know :)

**Comment by zeroKilo** [comment]
I found these infos by some guy who has already done alot research it seems:

https://github.com/LuigiBlood/64dd/wiki/Disk-Format-%28SDK%29
https://github.com/LuigiBlood/64dd/wiki/System-Area

and the old official n64 dev manuals:

http://n64devkit.square7.ch/ -> click on N64 Disk Drive

if you find some infos on how to read the data, I could go on, but well sofar I dont have enough information on how to read these files.

**Comment by LuEnCam** [comment]
Yeah LuigiBlood is definitely the 64DD Guru ! He’s the one behind most of the work under the Disk Drive scene. 

There’s also here the Disk Drive Programming Manual in case it could help : https://ultra64.ca/files/documentation/nintendo/Nintendo_64_Disk_Drive_Programmers_Guide_NUS-06-0121-001D.pdf

**Comment by zeroKilo** [comment]
thanks alot^^ I dont have time currently to look into it, but remind me in like half a year and I can maybe convert it into code :D

**Comment by LuEnCam** [comment]
Perfect thanks :) I also may have found the tool that was used to build the DD ROMs from the source code. If you want, we can try to communicate about this topic in an other place instead of here (of course, when you'll have the time).

**Comment by zeroKilo** [comment]
dunno if you are on discord, but you can find and dm me here most of the time: https://discord.gg/2Ek2F6AwDj
